source/grid/tria_accessor.cc

```diff
@@ -1082,6 +1082,102 @@ set (const internal::Triangulation::TriaObject<structdim> &object) const
 
 
 
+template <>
+TriaAccessor<0,1,1>
+TriaAccessor<1,1,1>::vertex_accessor (const unsigned int i) const
```

Comment:
This does not work. I get this error:
```error: invalid use of incomplete type `class dealii::TriaAccessor<1,1,spacedim>'```
```error: declaration of `class dealii::TriaAccessor<1,1,spacedim>'```

This looks similar to this: http://stackoverflow.com/questions/6138439/understanding-simple-c-partial-template-specialization